csro/oiwfs-adc-assembly/component-model.conf

```diff
@@ -1,8 +1,8 @@
 subsystem = IRIS
-component = oiwfs-adc-assembly
+component = oiwfs_adc_assembly
 
-prefix = iris.oiwfs.adc
-modelVersion = "1.0"
+//prefix = iris.oiwfs.adc
```

Comment:
is `//` commenting lines out? what is the reason?

Reply:
I think @abrighton was drawing our attention to the fact that prefix was going away. You're right though, we should get rid of these comments as part of this PR.

I'll sort out the merge conflicts first, then do another stand-alone commit removing comments.

Reply:
OK, one of my recent commits cleans up both these commented-out prefix lines, and updates the component name following Takashi's suggestion.